Patch to fix the definition of OPT_AUDIT to be octal instead of decimal,
so that it works properly in a bit field.

Authors: Corey Wright <undefined@pobox.com>

Upstream status: not yet submitted

--- Linux-PAM/modules/pam_tally/pam_tally.c~	2007-10-12 01:49:15.000000000 +0000
+++ Linux-PAM/modules/pam_tally/pam_tally.c	2007-10-12 03:26:04.000000000 +0000
@@ -95,7 +95,7 @@ struct tally_options {
 #define OPT_PER_USER			 010
 #define	OPT_NO_LOCK_TIME		 020
 #define OPT_NO_RESET			 040
-#define OPT_AUDIT                        100
+#define OPT_AUDIT                       0100
 
 
 /*---------------------------------------------------------------------*/
